legal-api/src/legal_api/services/authz.py

```diff
@@ -418,29 +451,38 @@ def has_blocker(business: Business, state_filing: Filing, allowable_filing: dict
     return False
 
 
-def has_business_blocker_default(blocker_checks: dict, has_default_blocker: bool):
+def has_business_blocker_default(blocker_checks: dict, has_default_blocker: dict):
```

Comment:
please rename `has_business_blocker_default` to `has_business_blocker`.

And also rename `has_default_blocker` to `has_business_blocker_dict`

Reply:
But "has" still implies it's a boolean. How about simply `business_blocker_dict` ?